.clang-format

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Differences from Chromium style:
* Set `AllowAllParametersOfDeclarationOnNextLine: true`
* Set `BinPackParameters: true`
* Removed `ForEachMacros`
* Tweaked `IncludeCategories` and `IncludeIsMainRegex`
* Removed `RawStringFormats`
* Set `Standard: Cpp11`
* Removed `StatementMacros`

Reply:
It makes sense to have two commits for this file: one adding original Chromium style, next one applying all the tweaks.